hack/update-bundles.sh

```diff
@@ -120,6 +126,14 @@ for b in $BUNDLES; do
     # Now push
     conftest_push $b "$push_repo:$tag"
 
+    # Add OCI annotations to the bundle image
+    tmp_oci_dir="$(mktemp -d --tmpdir)"
+    tmp_oci_dirs+=("${tmp_oci_dir}")
+    skopeo copy docker://"$push_repo:$tag" dir:"${tmp_oci_dir}"
+    manifest="$(jq -c '. += { "annotations": { "org.opencontainers.image.revision": "'"${last_update_sha}"'" } }' "${tmp_oci_dir}/manifest.json")"
```

Comment:
Not sure exactly what the jq is doing, but I'll assume it's right.

Reply:
`-c` is for compact output (not indented), and the rest is adding `annotations["org.opencontainers.image.revision"]` to the manifest. Kinda emphasizes that we need [HACBS-1958](https://issues.redhat.com/browse/HACBS-1958)